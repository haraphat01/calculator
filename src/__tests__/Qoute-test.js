import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavBar from '../components/Navbar';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <>
        <BrowserRouter>
          <NavBar />
        </BrowserRouter>
        <h1 style={{ textAlign: 'center' }}>De te fabula narratur!</h1>
      </>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
