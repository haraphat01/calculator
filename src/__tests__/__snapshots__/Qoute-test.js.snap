// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div>
    <a
      href="/"
      onClick={[Function]}
    >
      Home 
    </a>
    <a
      href="/App"
      onClick={[Function]}
    >
      Calculator 
    </a>
    <a
      href="/Quote"
      onClick={[Function]}
    >
      Quote 
    </a>
  </div>,
  <h1
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    De te fabula narratur!
  </h1>,
]
`;
